common/src/main/java/com/wynntils/features/user/ChatMentionFeature.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.user;
+
+import com.wynntils.core.config.Config;
+import com.wynntils.core.config.ConfigHolder;
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.handlers.chat.event.ChatMessageReceivedEvent;
+import com.wynntils.utils.mc.ComponentUtils;
+import com.wynntils.utils.mc.McUtils;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import net.minecraft.ChatFormatting;
+import net.minecraft.network.chat.Component;
+import net.minecraft.network.chat.MutableComponent;
+import net.minecraft.sounds.SoundEvents;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+public class ChatMentionFeature extends UserFeature {
+    @Config
+    public boolean markMention = true;
+
+    @Config
+    public boolean dingMention = true;
+
+    @Config
+    ChatFormatting rewriteColorCode = ChatFormatting.YELLOW;
+
+    @Config
+    public String aliases = "";
+
+    private Pattern pattern;
```

Comment:
You seem to have forgotten the constructor.

Reply:
I asked you to move the initialization to the constructor from the field declaration. You removed it from the field, but you never added it to the constructor. You need to add a no-args constructor and add a `pattern = buildPattern` to it.